account_invoice_update_wizard/views/account_invoice.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+<record id="invoice_supplier_form" model="ir.ui.view">
```

Comment:
Bad indentation.

Reply:
It is not important I think, and the code was like that before (not part of the migration). Often the indentation is done as if `<odoo>` tag was not there to gain line length.